examples/chip-tool/py_matter_chip_tool_adapter/matter_chip_tool_adapter/encoder.py

```diff
@@ -16,7 +16,88 @@
 import json
 import re
 
+_ANY_COMMANDS_LIST = [
+    'ReadById',
+    'WriteById',
+    'SubscribeById',
+    'ReadEventById',
+    'SubscribeEventById',
+    'ReadAll',
+    'SubscribeAll',
+]
+
+_ANY_COMMANDS_LIST_ARGUMENTS_WITH_WILDCARDS = [
+    'ClusterId',
+    'AttributeId',
+    'EventId',
```

Comment:
As a followup: we should allow wildcarding the endpoint too.  The wildcard value there is is 0xFFFF, though, not 0xFFFFFFFF.

Reply:
Agreed. I have not done it here mostly because endpoint is not a "command" specific argument and goes via a different path. But yes it does makes total sense.